<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderEntry</class>
 <widget class="BaseWnd" name="OrderEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrderEntry</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
      <item>
       <widget class="InstrumentSelector" name="InsSelector" native="true">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
       </widget>
      </item>
      <item>
       <widget class="OrderEntryCenterWnd" name="CenterWnd" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="OrderEntryRightPane" name="RightPane" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseWnd</class>
   <extends>QWidget</extends>
   <header>basewnd.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InstrumentSelector</class>
   <extends>QWidget</extends>
   <header location="global">InstrumentSelector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrderEntryCenterWnd</class>
   <extends>QWidget</extends>
   <header>OrderEntryCenterWnd.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrderEntryRightPane</class>
   <extends>QWidget</extends>
   <header>OrderEntryRightPane.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
